extensions/micrometer-opentelemetry/runtime/src/main/java/io/quarkus/micrometer/opentelemetry/runtime/MicrometerOtelBridgeRecorder.java

```diff
@@ -0,0 +1,48 @@
+package io.quarkus.micrometer.opentelemetry.runtime;
+
+import java.util.concurrent.TimeUnit;
+import java.util.function.Function;
+
+import jakarta.enterprise.inject.Instance;
+import jakarta.enterprise.util.TypeLiteral;
+
+import io.micrometer.core.instrument.Clock;
+import io.micrometer.core.instrument.MeterRegistry;
+import io.micrometer.core.instrument.Metrics;
+import io.opentelemetry.api.OpenTelemetry;
+import io.opentelemetry.instrumentation.micrometer.v1_5.OpenTelemetryMeterRegistry;
+import io.quarkus.arc.SyntheticCreationalContext;
+import io.quarkus.opentelemetry.runtime.config.runtime.OTelRuntimeConfig;
+import io.quarkus.runtime.annotations.Recorder;
+
+@Recorder
+public class MicrometerOtelBridgeRecorder {
+
+    public Function<SyntheticCreationalContext<Object>, Object> createBridge(OTelRuntimeConfig otelRuntimeConfig) {
```

Comment:
Let's not use `Object` here as we know the types

Reply:
right.